recipes/eth-utils/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "eth-utils" %}
+{% set version = "1.9.4" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 85f561096ebe34f18173e75f3062753a5a2464f7453df435a1ee846dfd58ce2e 
+
+build:
+  number: 0
+  skip: true   # [py>=40 or py2k]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools-markdown
+  run:
+    - cytoolz <1.0.0,>=0.10.1  # [py]
```

Comment:
I think `py` is always true

Reply:
so what is a selector that matches cpython but not pypy? (for the record, this is going off of setup.py, not tested myself so far)

Reply:
ok found it 
```
# [python_impl == 'cpython']
# [python_impl == 'pypy'] ```